vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -872,6 +879,7 @@ std::vector<std::string> AdsImpl::GetWinningCategories() {
 
   for (const auto& page_score : page_score_history) {
     DCHECK(page_score.size() == count);
+    DCHECK(user_model_);
```

Comment:
Are you sure we don't need a check on null pointer for `user_model_`? As that is the reason of the crash. Maybe we should fail with empty `winning_categories`?

Reply:
We've discussed this on a call, the place should never be reached when the pointer is null. Let's 
 apply the fix and reopen the issue if we still see it.